common/CommunityToolkit.Labs.Shared/Renderers/ToolkitSampleRenderer.xaml

```diff
@@ -113,13 +115,16 @@
                             </AppBarButton.Icon>
                         </AppBarButton>
 
-                        <!--  TO DO: Implement FlowDirection  -->
-                        <!--<AppBarButton Label="Toggle right-to-left"
+                        <!--  wasm:Visibility="Collapsed"  -->
```

Comment:
https://user-images.githubusercontent.com/9384894/184418929-412b13a8-a8ce-4e24-bd90-780742f1e25b.png

@niels9001 that seems weird that Visibility wouldn't be working here (@jeromelaban)? Does `wasm:IsEnabled="False"` work as an alternative?

Reply:
Using `wasm:Visibility` would generally require to put `wasm` in the ignorables so that UWP skips the declaration. That said, `wasm` is only available on heads, not libraries (libraries are cross targeted with wasm and skia, so they can't be discriminated), so it depends on where this code is included.